server/http.go

```diff
@@ -260,6 +260,7 @@ func (s *HTTPd) RegisterSystemHandler(site *core.Site, valueChan chan<- util.Par
 			"interval":           {"POST", "/interval/{value:[0-9.]+}", settingsSetDurationHandler(keys.Interval)},
 			"updatesponsortoken": {"POST", "/sponsortoken", updateSponsortokenHandler},
 			"deletesponsortoken": {"DELETE", "/sponsortoken", deleteSponsorTokenHandler},
+			"liveconfig":         {"GET", "/live", LiveConfigHandler(site)},
```

Comment:
Ich bin mit dem Namen noch nicht glücklich. Ich wäre dafür das in den `/system` Namespace zu hängen. Also `GET /system/config` analog zu `GET /system/log`.